respec/template.html

```diff
@@ -374,7 +372,7 @@ <h2>Scope</h2>
     and enables federated search across multiple sites using a uniform query mechanism and structure.
   </p>
   <p>
-    The namespace for DPROD terms is <a href="https://ekgf.github.io/dprod/#">https://ekgf.github.io/dprod/#</a>
```

Comment:
Was already updated, this leads me to be believe you made these changes on an older version of main

Reply:
I created my branch from main after our call Friday which included Natasa's changes.

Reply:
Various changes slipped into main since then, can't merge this PR until it's updated from main first